src/exported-tests/component-tests.ts

```diff
@@ -0,0 +1,39 @@
+import merge from "lodash-es/merge";
+
+class ComponentTests {
```

Comment:
Just want to double check if this class is still something we need to write out like this, or if there's an implementation we can use (or extend) from the `exported-tests` package?

Reply:
If not then I think we're good to go with this PR 😁

Reply:
If I'm not mistaken @elizabethsjudd, I don't think the file is available in the `exported-tests` package. If it is we definitely want to extend that one.

Reply:
This file is not yet opened source.  We were planning on putting it wherever we put the shared UI unit tests which would be separate from the Architecture (stuff to talk about at our meeting on Tuesday)